src/overlays/actors/ovl_Bg_Jya_Cobra/z_bg_jya_cobra.c

```diff
@@ -26,41 +52,630 @@ const ActorInit Bg_Jya_Cobra_InitVars = {
+    { 0x80, 0xA3, 0xA6, 0xA6, 0xA6, 0xA6, 0xA6, 0xA6, 0xA6, 0xA3, 0x80 },
+    { 0x80, 0xA3, 0xA6, 0xA6, 0xA6, 0xA6, 0xA6, 0xA6, 0xA6, 0xA3, 0x80 },
+    { 0x80, 0xA0, 0xA5, 0xA6, 0xA6, 0xA6, 0xA6, 0xA6, 0xA5, 0xA0, 0x80 },
+    { 0x20, 0x80, 0xA0, 0xA5, 0xA6, 0xA6, 0xA6, 0xA5, 0xA0, 0x80, 0x20 },
+    { 0x00, 0x20, 0x80, 0xA0, 0xA3, 0xA3, 0xA3, 0xA0, 0x80, 0x20, 0x00 },
+    { 0x00, 0x00, 0x20, 0x80, 0x80, 0x80, 0x80, 0x80, 0x20, 0x00, 0x00 },
+};
+
+u8 D_80897398[3][3] = {
+    { 0x20, 0x80, 0x20 },
+    { 0x80, 0xA0, 0x80 },
+    { 0x20, 0x80, 0x20 },
+};
+
+Vec3f D_808973A4[] = {
+    { -6.0f, 100.0f, 7.6f },
+    { -12.6f, 69.200005f, -10.0f },
+    { -9.0f, 43.0f, -1.0f },
+    { -3.0f, 15.0f, 8.6f },
+    { -8.6f, 15.0f, 13.5f },
+    { -6.6f, 26.0f, 11.6f },
+    { -12.5f, 43.0f, 8.0f },
+    { -17.2f, 70.0f, 0.6f },
+    { -8.0f, 100.0f, 7.6f },
+    { 6.0f, 100.0f, 7.6f },
+    { 12.6f, 69.200005f, -10.0f },
+    { 9.0f, 43.0f, -1.0f },
+    { 3.0f, 15.0f, 8.6f },
+    { 8.6f, 15.0f, 13.5f },
+    { 6.6f, 26.0f, 11.6f },
+    { 12.5f, 43.0f, 8.0f },
+    { 17.2f, 70.0f, 0.6f },
+    { 8.0f, 100.0f, 7.6f },
+    { 0.0f, 70.0f, -11.3f },
+    { 0.0f, 44.6000023f, -2.0f },
+    { 0.0f, 15.0f, 10.6f },
+    { 0.0f, 15.0f, 0.3f },
+    { 0.0f, 26.0f, 11.6f },
+    { 0.0f, 88.4f, -1.4f },
+    { 0.0f, 95.700005f, 14.900001f },
+    { 0.0f, 101.4f, 5.0f },
+};
+
+Vec3f D_808974DC[] = {
+    { 12.0f, 21.3000011f, -2.5f },  { 30.0f, 21.3000011f, -2.5f }, { -15.0f, 21.3000011f, -2.5f },
+    { -30.0f, 21.3000011f, -2.5f }, { 12.0f, 21.3000011f, -2.5f },
+};
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Jya_Cobra/func_808959C4.s")
+s32 D_80897518[] = { 0x00000080, 0x000000A0, 0x000000A0, 0x00000080 };
 
+extern InitChainEntry D_80897528[];
+// D_80897528
+InitChainEntry D_80897528[] = {
+    ICHAIN_VEC3F_DIV1000(scale, 100, ICHAIN_CONTINUE),
+    ICHAIN_F32(uncullZoneForward, 1000, ICHAIN_CONTINUE),
+    ICHAIN_F32(uncullZoneScale, 800, ICHAIN_CONTINUE),
+    ICHAIN_F32(uncullZoneDownward, 1000, ICHAIN_STOP),
+};
+
+Vec3s D_80897538 = { 0, 0xC000, 0 };
+
+Vec3s D_80897540 = { 0, 0x4000, 0 };
+
+Vec3f D_80897548[] = {
+    { 0.1f, 0.1f, 0.1f },
+    { 0.072f, 0.072f, 0.072f },
+    { 0.1f, 0.1f, 0.132f },
+};
+
+void func_808958F0(Vec3f* arg0, Vec3f* arg1, f32 arg2, f32 arg3) {
+    arg0->x = (arg1->z * arg2) + (arg1->x * arg3);
+    arg0->y = arg1->y;
+    arg0->z = (arg1->z * arg3) - (arg1->x * arg2);
+}
+
+void func_8089593C(BgJyaCobra* this, GlobalContext* globalCtx, void* arg2, DynaPolyMoveFlag flags) {
+    s32 padding;
+    s32 sp30;
+    s32 padding2;
+
+    sp30 = 0;
+    DynaPolyInfo_SetActorMove(&this->dyna, flags);
+    DynaPolyInfo_Alloc(arg2, &sp30);
+    this->dyna.dynaPolyId = DynaPolyInfo_RegisterActor(globalCtx, &globalCtx->colCtx.dyna, &this->dyna.actor, sp30);
+    if (this->dyna.dynaPolyId == 50) {
+        // Warning : move BG Registration Failure
```

Comment:
i believe theres a define for the 50 value now, check out similar functions in other actors (search for the warning string, this function is common)